frontend/src/static/js/main.js

```diff
@@ -21,7 +22,8 @@ window.app = new window.Vue({
     users: [],
     userUpdated: false,
 
-    isLoading: false,
+    isLoadingOverlayEnabled: false,
```

Comment:
Seems that the implementation can be simplified a bit? We are already using the same spinner, why we cannot only have one attribute for its state and another attribute for the text below it? Will it be more elegant in the implementation? And we can extend it to any scenario that needs a loading overlay?

Reply:
Done